function-proto/src/main/proto/function.proto

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+syntax = "proto3";
+import "fntypes.proto";
+option java_multiple_files = false;
+option java_package = "io.sk8s.sidecar.grpc.function";
+option java_outer_classname = "FunctionProtos";
+
+package function;
+
+service StringFunction {
+  rpc Call(fntypes.Request) returns (fntypes.Reply) {}
+}
```

Comment:
Newline

Reply:
I'm not able to fix this for some reason